libcrypt/PKGBUILD

```diff
@@ -4,6 +4,7 @@ pkgbase=libcrypt
 pkgname=('libcrypt' 'libcrypt-devel')
 pkgver=2.1
 pkgrel=1
+epoch=1
```

Comment:
Git for Windows does not modify `libcrypt` Therefore this change should be unnecessary.